public/addEditMovie.js

```diff
@@ -0,0 +1,121 @@
+import { enableInput, inputEnabled, message, setDiv, token } from "./index.js";
+import { showMovie } from "./movie.js";
+
+let addEditDiv = null;
+let director = null;
+let title = null;
+let genre = null;
+let addingMovie = null;
+
+export const handleAddEditM = () => {
```

Comment:
I like that you've delineated these, and it's clear enough to me. When working in larger projects you'll likely want to give more descriptive names (i.e. handleAddEditMusic vs. handleAddEditMovie or something) so people can quickly tell what a function should do by its name